index.html

```diff
@@ -281,7 +282,39 @@ <h2>Skills</h2>
         </div>
 
     </section>
+    <div class="div"><br><br><br></div>
+    <!-- Contact Us Section -->
+    <div class="div">
+        <section class="contact">
+            <div class="contact-items">
+                <h2>Contact me</h2>
+            </div>
+            <div class="contact-items">
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
 
+            </div>
+            <div class="contact-items">
+                <form action="https://formspree.io/f/{form_id}" method="post" class="form-flex">
+                    <div class="form-flex-item ">
```

Comment:
- [ ] I would suggest you to kindly All input fields  in a single `form` element.this will indicate a proper sementic html